client/src/types/trackLyrics.ts

```diff
@@ -2,10 +2,15 @@ interface ILyricDetails {
   content: string;
   artist: string;
   title: string;
-  geniusUrl: string;
+  syncedLyricsArray?: Array<ILRCContent>;
 }
 
 export interface ITrackLyrics {
   currentlyPlayingItemId: string;
   lyrics: ILyricDetails | undefined;
 }
+
+export interface ILRCContent {
+  content: string;
+  timestamp: number;
+}
```

Comment:
You have this in `client\src\types\trackLyrics.ts` - can reference that here and drop this